examples/configs/pose_graph/pose_graph_synthetic.yaml

```diff
@@ -10,16 +11,15 @@ loop_closure_outlier_ratio: 0.25
 dataset_size: 256 
 batch_size: 128
 
-
 inner_optim:
   optimizer_cls: LevenbergMarquardt
+  linear_solver_cls: LUCudaSparseSolver
```

Comment:
Wonder if we should change the default solver to be `BaSpaCho` since it is much faster.

Reply:
This is set to this one because Baspacho is an optional install, so this should work for a larger set of users by default.